src/app/containers/CpsMetadata/utils.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable no-unused-vars */
+// eslint-disable-next-line import/prefer-default-export
+export const getBrandedImage = (image, service) => {
+  // const imageUrl = image
+  //   .split('/')
+  //   .splice(3)
+  //   .join('/');
+  // return `http://ichef.bbci.co.uk/news/1024/branded_${service}/${imageUrl}`;
```

Comment:
I think the image size is included in the AReS response, so we should be able to use it instead of hardcoding.

Reply:
Branded images need to be 1024, which is a requirement for open graph images as well: 

[Internal link to slack chat](https://bbcnews.slack.com/archives/CK85Q5SL9/p1579532762027200?thread_ts=1579531746.026900&cid=CK85Q5SL9)

Reply:
we need to specify this size based on what twitter / facebook etc expects - not what ARES returns

Reply:
So should we just hardcode rather than using the data from AReS? I guess if the PAL redirects are, it makes sense for us to do it too.

Reply:
Yeah, I will confirm this from Chris